lib/events.js

```diff
@@ -36,6 +36,7 @@ EventEmitter.usingDomains = false;
 
 EventEmitter.prototype.domain = undefined;
 EventEmitter.prototype._events = undefined;
+EventEmitter.prototype._eventsCount = 0;
```

Comment:
You might check the performance on this, but I remember at least back with Crankshaft, putting properties like this (that are likely to change in every `EventEmitter` instance) on the prototype could slow things down.

Reply:
It's definitely fine now from the testing I did. (I ran benchmarks with & without.)